src/diffpy/utils/transforms.py

```diff
@@ -16,9 +16,7 @@
     "The supplied q-array and wavelength will result in an impossible two-theta. "
     "Please check these values and re-instantiate the DiffractionObject with correct values."
 )
-invalid_input_emsg = (
-    "Input values have resulted in an infinite output. Please ensure there are no zeros in the input."
-)
+inf_output_msg = "WARNING: The largest output is infinite and cannot be plotted."
```

Comment:
@sbillinge out of curiosity, 

for both error and warning messages, do we still want to maintain our convention of 1) reason for error, (2) what to do to fix it ?

Reply:
I tweaked this a bit and downgraded it to an INFO from a WARNING....don't want users to freak out....